src/components/application_manager/include/application_manager/app_launch/app_launch_ctrl_impl.h

```diff
@@ -77,6 +78,7 @@ class AppLaunchCtrlImpl : public AppLaunchCtrl {
 
   AppsLauncher apps_launcher_;
   DeviceAppsLauncher device_apps_launcher_;
+  sync_primitives::Lock launch_ctrl_lock_;
```

Comment:
```suggestion
  sync_primitives::Lock app_launcher_lock_;
```
possibly a better name? I don't have too strong of an opinion though.

Reply:
Done